app/javascript/packs/active_admin/print.scss

```diff
@@ -0,0 +1,2 @@
+/* Active Admin Print Stylesheet */
+@import "~@activeadmin/activeadmin/src/scss/print";
```

Comment:
SCSS should be served by the asset pipeline, not by webpacker.

JS -> webpacker
CSS -> asset pipeline

that's the Rails default

Reply:
This was generated by the ActiveAdmin, moving to stylesheets.